configure.ac

```diff
@@ -3,7 +3,7 @@ AC_PREREQ(2.59)
 AC_INIT(alsa-utils, 1.2.5.1)
 AC_CONFIG_SRCDIR([aplay/aplay.c])
 AC_PREFIX_DEFAULT(/usr)
-AM_INIT_AUTOMAKE
+AM_INIT_AUTOMAKE([subdir-objects])
```

Comment:
Can we say why we need this in teh commit message.

Reply:
I dont think you need to do this here, configure.ac should be able to individually list the subdirs (there is another macro for it)